posix.mak

```diff
@@ -139,6 +139,10 @@ endif
 ifdef ENABLE_COVERAGE
 DFLAGS  += -cov
 endif
+ifneq (,$(TZ_DATABASE_FILE_DIR))
+DFLAGS += -version=TZDatabaseDir -J$(TZ_DATABASE_FILE_DIR)
```

Comment:
Hmm that doesn't look very elegant, but I'm also in lack of a better idea. Have you considered defining `version=NixOS`?

Reply:
NixOS is to specific I think. Guix will have the same problems at least.

Reply:
But this would still require you to generate the file? How about generating/writing this file to $GENERATED directly in the Makefile? I guess that would save you one step and would make the docs easier as it's then just `TZ_DATABASE_DIR` what needs to be passed.